File: Spring_Study/spring-jwt/src/main/java/com/example/springjwt/jwt/JwtProperties.java
package com.example.springjwt.jwt;

public interface JwtProperties {
    String SECRET = "siwonpark"; // 비밀 키 값
    int EXPIRATION_TIME = 172800000; // 2일
    String TOKEN_PREFIX = "Bearer ";
    String HEADER_STRING = "Authorization";

}

File: Kotlin_Study/kotlin-security/src/main/kotlin/com/example/kotlinsecurity/util/JwtProperties.kt
package com.example.kotlinsecurity.util

interface JwtProperties {

    companion object {
        const val ISSUER: String = "siwonpark"
        const val SECRET: String = "dyAeHubOOc8KaOfYB6XEQoEj1QzRlVgtjNL8PYs1A1tymZvvqkcEU7L1imkKHeDa"
        const val TOKEN_PREFIX: String = "Bearer "
        const val HEADER_STRING: String = "Authorization"
        const val ACCESS_TOKEN_EXPIRATION: Long = 60 * 60 * 1000L // 1시간 설정
        const val REFRESH_TOKEN_EXPIRATION: Long = 2 * 60 * 60 * 1000L // 2시간 설정
    }

}